app/scripts/directives/istagSelect.js

```diff
@@ -35,9 +35,7 @@ angular.module("openshiftConsole")
         $scope.isNamesByNamespace = {};
 
         // Check if the istag object contains data about namespace/imageStream/tag so the ui-select will be pre-populated with them
-        var shouldPrepopulate = _.has($scope, 'istag.namespace') &&
-                                _.has($scope, 'istag.imageStream') &&
-                                _.has($scope, 'istag.tagObject.tag');
+        var shouldPrepopulate = (_.get($scope, 'istag.namespace') && _.get($scope, 'istag.imageStream') && _.get($scope, 'istag.tagObject.tag')) ? true : false;
```

Comment:
But that's OK. A non-empty string is truthy in JavaScript.

https://developer.mozilla.org/en-US/docs/Glossary/Truthy